--- Setup-Mystra.tp2
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Setup-Mystra.tp2"	
@@ -47,12 +47,12 @@ ADD_KIT ~MYSTRA~
 // Nom du kit en lettres minuscules et mixtes, et description du kit, ajoutés à KITLIST.2da
 SAY ~prêtre de mystra~
 SAY ~Prêtre de Mystra~
-SAY ~PRETRE DE MYSTRA: disciple de la déesse de la Magie et des possibilités. Gardienne de la toile magique qui englobe les royaumes, Mystra est respecté par tout les lanceurs de sort, ce qui fait d'elle l'une des plus puissantes divinités des Royaumes.
+SAY ~PRÊTRE DE MYSTRA: disciple de la déesse de la Magie et des possibilités. Gardienne de la toile magique qui englobe les royaumes, Mystra est respectée par tout les lanceurs de sort, ce qui fait d'elle l'une des plus puissantes divinités des Royaumes.
 
 Avantages :
-- Gain de 1 % de Résistance magique par niveau
-- Lame noire du désastre une fois par jour tout les 10 niveaux à partir du niveau 20.
-- Déviation de sorts une fois par jour tout les 5 niveaux à partir du niveau 10.
+- Gain de 1 % de Résistance à la magie par niveau
+- Lame noire du désastre une fois par jour tous les 10 niveaux à partir du niveau 20.
+- Déviation de sorts une fois par jour tous les 5 niveaux à partir du niveau 10.
 
 Désavantages :
 - aucun~
@@ -64,7 +64,7 @@ SAY NAME1 ~Amulette~
 
 SAY UNIDENTIFIED_DESC ~Normalement elle est automatiquement identifiée...~
 
-SAY NAME2 ~La Fois de Mystra~
+SAY NAME2 ~La Foi de Mystra~
 
 SAY DESC ~Cette amulette accueillie une partie de l'essence divine de Mystra lors du temps des troubles.
 
@@ -74,7 +74,7 @@ PARAMETRES
 
 Capacités d'équipement :
 Mémorisation de 1 sort additionnel du 6ème et du 7ème niveau.
-+10 % de Résistance magique.
++10 % de Résistance à la magie.
 Intelligence +1.
 
 Utilisable seulement par :
