--- Lysre\French\BCKLysre.tra
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Lysre\\French\\BCKLysre.tra"	
@@ -1,190 +1,190 @@
 //.////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //BANTERS AVEC CHARNAME   ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1 : 1xxx
-@1000    = ~*Chose rare, Lysre s'approche de vous, volant tranquillement en cercle autour de votre crane*
-Dites moi <CHARNAME>? Si j'en crois la vision que m'a accordee la Grande Tiamat le jour de notre rencontre, vous etes un enfant de Bhaal. Par pure curiosite, vous me laisseriez goutter votre sang... votre essence? Ce n'est pas tout les jours qu'on a la possibilite de goutter une essence divine presque a la source. ~
+//N?1 : 1xxx
+@1000    = ~*Chose rare, Lysre s'approche de vous, volant tranquillement en cercle autour de votre crâne*
+Dites moi <CHARNAME>? Si j'en crois la vision que m'a accordée la Grande Tiamat le jour de notre rencontre, vous êtes un enfant de Bhaal. Par pure curiosité, vous me laisseriez goutter votre sang... votre essence? Ce n'est pas tous les jours qu'on a la possibilité de goûter une essence divine presque à la source. ~
 
 @1001    = ~Tu veux boire mon sang ?~
-@1002    = ~Oh, n'ayez pas cet air de jeune bipede a ma vue. Ce n'est presque rien tres cher, juste quelques gouttes, vous ne sentirez meme pas mes crocs. Je vous l'assure, je sais me montrer aussi douce que la brume.~
+@1002    = ~Oh, n'ayez pas cet air de jeune bipède à ma vue. Ce n'est presque rien très cher, juste quelques gouttes, vous ne sentirez même pas mes crocs. Je vous l'assure, je sais me montrer aussi douce que la brume.~
 
-@1003    = ~La reponse est non~
+@1003    = ~La réponse est non~
 
-@1004    = ~Si ca t'amuse Lysre.~
+@1004    = ~Si ça t'amuse Lysre.~
 
-@1005    = ~*Alors que vous tendez la main, Lysre se pose delicatement sur votre bras. La pression de ses griffes est ferme contre vos chair, sans pour autant les meurtirs. Lentement, la dragonne avancant son museau jusqu'a votre main.* ~
+@1005    = ~*Alors que vous tendez la main, Lysre se pose délicatement sur votre bras. La pression de ses griffes est ferme contre vos chairs, sans pour autant les meurtrir. Lentement, la dragonne avançant son museau jusqu'à votre main.* ~
 @1006    = ~*Reculer la main*~
 @1007    = ~*Laisser faire*~
 
-@1008    = ~*Avec une lenteur manifestement exageree, Lystre entrouve sa gueule, laissant paraitre des crocs luisants. Elle fait mine de renifler lentement la peau, avant de planter ses crocs dans votre main. Etrangement vous ne ressentez aucune douleur. Juste une desagreable sensation de froid se rependant lentement dans votre chair.*~
-@1009    = ~*Ca suffit Lysre!*~
+@1008    = ~*Avec une lenteur manifestement exagérée, Lysre entrouvre sa gueule, laissant paraître des crocs luisants. Elle fait mine de renifler lentement la peau, avant de planter ses crocs dans votre main. Étrangement, vous ne ressentez aucune douleur. Juste une désagréable sensation de froid se répandant lentement dans votre chair.*~
+@1009    = ~*Ça suffit Lysre!*~
 @1010    = ~*Laisser faire*~
 
-@1011    = ~*Tout a coup, vous sentez une violente douleur vous prendre au coeur, votre vision se noircie brutalement et vos jambes commencent a trembler.
-Puis tout aussi rapidement, la douleur cesse, mis a part la legere trace des crocs de Lysre sur votre peau, rien ne temoigne de cette sensation glaciale.* ~
-@1012    = ~Qu'est ce que c'etait !*~
+@1011    = ~*Tout à coup, vous sentez une violente douleur vous prendre au coeur, votre vision se noircie brutalement et vos jambes commencent à trembler.
+Puis tout aussi rapidement, la douleur cesse, mis à part la légère trace des crocs de Lysre sur votre peau, rien ne témoigne de cette sensation glaciale.* ~
+@1012    = ~Qu'est ce que c'était !*~
 @1013    = ~Lysre !~
-@1014    = ~*S'eloigner*~
+@1014    = ~*S'éloigner*~
 
-@1015    = ~*Lysre sourit, elle a repris ses distances et se tortille* N'ayez pas peur <CHARNAME>, je n'ai pris qu'un peu de sang et une etincelle de votre ame. Rien qui ne peut vous nuire comme le veut le contrat. C'etait delicieux, une saveur de charogne et de mort, de carnages et de sang. C'etait... (ricane) Divin.~
+@1015    = ~*Lysre sourit, elle a repris ses distances et se tortille* N'ayez pas peur <CHARNAME>, je n'ai pris qu'un peu de sang et une étincelle de votre âme. Rien qui ne peut vous nuire comme le veut le contrat. C'était délicieux, une saveur de charogne et de mort, de carnages et de sang. C'était... (ricane) Divin.~
 @1016    = ~Ne recommence jamais !~
 @1017    = ~Mais pourquoi !~
 @1018    = ~L'ignorer et partir~
 
-@1019    = ~*Lysre sourit* Cela m'est impossible de vous prelever une etincelle de plus, je ne peux vous nuir <CHARNAME>, c'est dans notre contrat. Et ma parole est absolue (ricane)~
-@1020    = ~Meme si celle de Bhaal est infame selon l'avis de beaucoup. Elle a le gout de la mort, et de la putrefaction... Pourquoi me priver d'un peu d'essence de Divinite dotee d'une telle saveur? C'est un plat de choix.~
-@1021    = ~Merci encore <CHARNAME>! Votre generosite vous honore (ricane)~
+@1019    = ~*Lysre sourit* Cela m'est impossible de vous prélever une étincelle de plus, je ne peux vous nuir <CHARNAME>, c'est dans notre contrat. Et ma parole est absolue (ricane)~
+@1020    = ~Même si celle de Bhaal est infâme selon l'avis de beaucoup. Elle a le goût de la mort, et de la putréfaction... Pourquoi me priver d'un peu d'essence de Divinité dotée d'une telle saveur ? C'est un plat de choix.~
+@1021    = ~Merci encore <CHARNAME>! Votre générosité vous honore (ricane)~
 
-//interuption
+//interruption
 @1022    = ~Je n'ai pas encore atteint l'essence <CHARNAME>. Laissez moi poursuivre je vous prie.~
 @1023    = ~D'accord, mais fait vite!~
-@1024    = ~Non, ca suffit !~
+@1024    = ~Non, ça suffit !~
 
 //Refus
 @1025    = ~Je note...<CHARNAME> N'attendez rien de moi, si je ne peux rien attendre de vous.~
 
-//N°2
+//N?2
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //BANTERS AVEC NPC        ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 //Keldorn/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//Aucun dialogue prevu, personnages incompatibles
+//Aucun dialogue prévu, personnages incompatibles
 
 //Imoen   2100///////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2101  = ~C'est donc toi que <CHARNAME> a secouru envers en contre tous...~
+//N?1
+@2101  = ~C'est donc toi que <CHARNAME> a secouru envers et contre tous...~
 @2102  = ~Et toi tu es son dragon de poche ! Comme c'est mignon !~
-@2103  = ~(Crache) SILENCE ! Je n'appartient a personne, gamine ! Je devrais t'arracher la langue pour que tu evites de proferer de telles insultes!~
-@2104  = ~Ah, detend toi Lysre, je blague !~
-@2105  = ~Remercie <CHARNAME>, sans sa presence ici, tu serais deja morte.~
+@2103  = ~(Crache) SILENCE ! Je n'appartiens à personne, gamine ! Je devrais t'arracher la langue pour que tu évites de proférer de telles insultes!~
+@2104  = ~Ah, détend toi Lysre, je blague !~
+@2105  = ~Remercie <CHARNAME>, sans sa présence ici, tu serais déjà morte.~
 @2106  = ~Tu es aussi acide que ta bave Lysre, pourtant tu es adorable !~
-@2107  = ~(Craque des machoires et souffle une brume fine en direction d'Imoen)~
-@2108  = ~Je suis sure que c'est une marque d'affection... Je t'aime aussi Lysre !~
-@2109  = ~*Lysre s'éloigne en sifflant, entouree de sa brume de poison*~
+@2107  = ~(Craque des mâchoires et souffle une brume fine en direction d'Imoen)~
+@2108  = ~Je suis sûre que c'est une marque d'affection... Je t'aime aussi Lysre !~
+@2109  = ~*Lysre s'éloigne en sifflant, entourée de sa brume de poison*~
 
-//N°2
+//N?2
 @2110  = ~(Je comprends mieux...)~
-@2111  = ~Oh Lysre, toujours aussi... mignonne a me surveiller en silence ?~
-@2112  = ~(C'est cet ecran de naivete... Ces sourires... En fait c'est comme une armure...)~
-@2113  = ~Attends, tu ne m'ecoutes pas en fait ? Houhou? Lysre?~
-@2114  = ~(L'essence de Bhaal devrait pourtant te rendre plus "interessante"... Presque aussi supportable que <CHARNAME>...)~
-@2115  = ~(Murmure)Tu sais, on dirais que tu veux me mordre la. Lysre? Y'a quelqu'un.~
-@2116  = ~(soupire) Quel gachis.~
+@2111  = ~Oh Lysre, toujours aussi... mignonne à me surveiller en silence ?~
+@2112  = ~(C'est cet écran de naïveté... Ces sourires... En fait c'est comme une armure...)~
+@2113  = ~Attends, tu ne m'écoutes pas en fait ? Houhou? Lysre?~
+@2114  = ~(L'essence de Bhaal devrait pourtant te rendre plus "intéressante"... Presque aussi supportable que <CHARNAME>...)~
+@2115  = ~(Murmure)Tu sais, on dirait que tu veux me mordre là. Lysre? Y'a quelqu'un.~
+@2116  = ~(soupire) Quel gâchis.~
 @2117  = ~De... Mais de quoi tu parles.~
-@2118  = ~*Lysre s'eloigne sans un mot*~
+@2118  = ~*Lysre s'éloigne sans un mot*~
 @2119  = ~C'est... Toujours un plaisir de discuter avec toi Lysre.~
 
 //Viconia  2200///////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
+//N?1
 @2200  = ~*Lysre s'approche de Viconia, avec pour une fois un semblant de respect dans le mouvement*~
-@2201  = ~Tu es une Drow interessante, Viconia, tu dois supporter ces bipedes de la surface, alors qu'ils te meprisent tous.~
-@2202  = ~Je n'ai pas eu le choix de quitter l'Ombreterre. La plupart des surfaciens me meprisent, oui, mais je m'en moque.~
-@2203  = ~La plupart... Je suppose que tu parles de <CHARNAME> c'est vrai que l'Enfant de Bhaal est un peu... a part.~
-@2204  = ~Effectivement. <CHARNAME> m'a sauvee deux fois la vie, quand des habitants de la surface voulaient ma mort juste a cause de mes origines. Je lui rendrai la pareille.~
-@2205  = ~Comme je disais, une Drow interessante. <CHARNAME> sais bien s'entourer.~
+@2201  = ~Tu es une Drow intéressante, Viconia, tu dois supporter ces bipèdes de la surface, alors qu'ils te méprisent tous.~
+@2202  = ~Je n'ai pas eu le choix de quitter l'Ombreterre. La plupart des surfaciens me méprisent, oui, mais je m'en moque.~
+@2203  = ~La plupart... Je suppose que tu parles de <CHARNAME> c'est vrai que l'Enfant de Bhaal est un peu... à part.~
+@2204  = ~Effectivement. <CHARNAME> m'a sauvée deux fois la vie, quand des habitants de la surface voulaient ma mort juste à cause de mes origines. Je lui rendrai la pareille.~
+@2205  = ~Comme je disais, une Drow intéressante. <CHARNAME> sait bien s'entourer.~
 
 //Aerie   2300////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
+//N?1
 @2300   = ~Dis donc, la gallinule, je me pose une question...~
-@2301   = ~Que... La gallinule? C'est a moi que tu parles?~
-@2302   = ~Evidemment !Tu es une elfe incapable de voler ! (Ricane) D'ailleurs, ma question : Ca te fait quoi d'etre clouee au sol ?~
+@2301   = ~Que... La gallinule? C'est à moi que tu parles?~
+@2302   = ~Évidemment ! Tu es une elfe incapable de voler ! (Ricane) D'ailleurs, ma question : Ça te fait quoi d'être clouée au sol ?~
 @2303   = ~(Hoquet) Je... M... Laisse moi ! ~
-@2304   = ~*Tout en ricanant, la dragonne s'amuse a suivre un peu Aerie avant de se lasser et de retourner a ses occupations*~
+@2304   = ~*Tout en ricanant, la dragonne s'amuse à suivre un peu Aérie avant de se lasser et de retourner à ses occupations*~
 
 //Minsc   2400//////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2400  = ~Minsc, ca fait un moment que tu me fixes. Tu veux perdre un oeil? ~
-@2401  = ~Bouh a le poil tout herisse a cause de toi. Voit on dirait un herisson!~
-@2402  = ~Et bien il faut reconnaitre que ton rongeur a plus d'intelligence que la moyenne des bipedes que j'ai croise, son maitre y compris~
+//N?1
+@2400  = ~Minsc, ça fait un moment que tu me fixes. Tu veux perdre un oeil? ~
+@2401  = ~Bouh a le poil tout hérissé à cause de toi. Voit, on dirait un hérisson!~
+@2402  = ~Et bien il faut reconnaître que ton rongeur a plus d'intelligence que la moyenne des bipèdes que j'ai croisé, son maître y compris~
 @2403  = ~(Squiik !)~
 @2404  = ~Shhhhhhhh!~
 @2405  = ~Il ne manque pas de cran. Je l'aime bien, je le mangerai plus tard.~
-@2406  = ~Bouh n'est pas a manger ! C'est un compagnon !~
-@2407  = ~(Soupire) Des mucles oui, pour le reste...~
+@2406  = ~Bouh n'est pas à manger ! C'est un compagnon !~
+@2407  = ~(Soupire) Des muscles oui, pour le reste...~
 
 //Edwin  2500//////////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2500  = ~Je dois reconnaitre, que pour un bipede, tes talents de sorcier sont impressionnant Edwin.~
-@2501  = ~Effectivement, je suis le meilleurs, tout simplement (Je le savais, elle est tombee sous mon charme ravageur, bientot j'en ferrai mon familier)~
+//N?1
+@2500  = ~Je dois reconnaître, que pour un bipède, tes talents de sorcier sont impressionnant Edwin.~
+@2501  = ~Effectivement, je suis le meilleur, tout simplement (Je le savais, elle est tombée sous mon charme ravageur, bientôt j'en ferai mon familier)~
 @2502  = ~*Fixe avec attention le sorcier, la gueule entrouverte*~
 @2503  = ~O-Oui Lysre ? (Elle me regarde de travers. <CHARNAME> ! Venez calmer votre animal !)~
-@2504  = ~*Ricane* Tu as de la chance de m'avoir croisee apres <CHARNAME> J'espere qu'il se debarassera de toi un jour...~
-@2505  = ~AH ! Impossible ! (Il est bien trop idiot pour pouvoir se passer de ma presence)~
+@2504  = ~*Ricane* Tu as de la chance de m'avoir croisée après <CHARNAME> J'espère qu'il se débarrassera de toi un jour...~
+@2505  = ~AH ! Impossible ! (Il est bien trop idiot pour pouvoir se passer de ma présence)~
 
-//Haerdalis  26xx /////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2600  = ~Tu as l'odeur d'un autre plan sur toi. J'aime beaucoup, ca me rapelle les terres de la grande Tiamat.~
-@2601  = ~Effectivement, je ne suis pas nee sur ce plan. Ton odorat est impressionnant Lysre.~
-@2602  = ~Tu n'es pas inutile a la difference de beaucoup, tu pourrais composer des poemes sur ma grandeur...~
-@2603  = ~Euh... oui, si je trouve le temps pour ca.~
+//Haer'dalis  26xx /////////////////////////////////////////////////////////////////////////////////////////////////
+//N?1
+@2600  = ~Tu as l'odeur d'un autre plan sur toi. J'aime beaucoup, ça me rappelle les terres de la grande Tiamat.~
+@2601  = ~Effectivement, je ne suis pas né sur ce plan. Ton odorat est impressionnant Lysre.~
+@2602  = ~Tu n'es pas inutile à la différence de beaucoup, tu pourrais composer des poèmes sur ma grandeur...~
+@2603  = ~Euh... oui, si je trouve le temps pour ça.~
 
 //Jan   27xx ///////////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
+//N?1
 @2700  = ~*Pour une fois, Lysre semble paisible.~
 @2701  = ~Une...~
-@2702  = ~SILENCE VERMINE. Au premier mot qui sort de cette gorge, et je te l'ouvre et je t'arrache les yeux. J'en ai ASSEZ de tes histoires Gnome.~
+@2702  = ~SILENCE VERMINE. Au premier mot qui sort de cette gorge, je te l'ouvre et je t'arrache les yeux. J'en ai ASSEZ de tes histoires Gnome.~
 
 //Jaheira 28xx  ///////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2800  = ~Lysre ! Ton existence nuit a l'equilibre ! Je ne comprend pas que <CHARNAME> te laisse en vie !~
-@2801  = ~L'equilibre est une foutaise de Druide. Seule compte la destruction au nom de Tiamat. Mais c'est tres complique pour une bipede de ta stature.~
-@2802  = ~Comment peut tu...~
-@2803  = ~Shhh, fait comme moi, Bipede, tolere les caprices de <CHARNAME>. Ca t'evitera des ennuis une fois liberee de sa protection.~
-@2804  = ~C'est toi qui beneficie de sa protection, Dragonne.~
+//N?1
+@2800  = ~Lysre ! Ton existence nuit à l'équilibre ! Je ne comprends pas que <CHARNAME> te laisse en vie !~
+@2801  = ~L'équilibre est une foutaise de Druide. Seule compte la destruction au nom de Tiamat. Mais c'est très compliqué pour une bipède de ta stature.~
+@2802  = ~Comment peux-tu...~
+@2803  = ~Shhh, fait comme moi, Bipède, tolère les caprices de <CHARNAME>. Ça t'évitera des ennuis une fois libérée de sa protection.~
+@2804  = ~C'est toi qui bénéficie de sa protection, Dragonne.~
 
 //Nalia   29xx ///////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@2900  = ~Lysre, n'a tu jamais connu la pitie?~
-@2901  = ~*La dragonne semble se figer en l'air* La pitie?~
-@2902  = ~Oui, l'envie de proteger les faibles.~
-@2903  = ~Prot... Mon dieu cette bipede est serieuse ? (Ricane) La pitie c'est un concept concu par les faibles pour les faibles. Je n'ai aucune pitie pour personne, ce monde est regit par la loi du plus puissant, au sommet se trouve Tiamat, puis viennent les dragons. Les bipedes peuvent au mieux nous servir ou nous nourrir avec leurs entrailles.~
+//N?1
+@2900  = ~Lysre, n'as-tu jamais connu la pitié ?~
+@2901  = ~*La dragonne semble se figer en l'air* La pitié?~
+@2902  = ~Oui, l'envie de protéger les faibles.~
+@2903  = ~Prot... Mon dieu cette bipède est sérieuse ? (Ricane) La pitié c'est un concept conçu par les faibles pour les faibles. Je n'ai aucune pitié pour personne, ce monde est régi par la loi du plus puissant, au sommet se trouve Tiamat, puis viennent les dragons. Les bipèdes peuvent au mieux nous servir ou nous nourrir avec leurs entrailles.~
 @2904  = ~Tu n'es qu'un monstre.~
-@2905  = ~Et toi une sotte ! Je te derange pourtant? Non j'ai la politesse de t'ignorer. Laisse moi tu veux.~
+@2905  = ~Et toi une sotte ! Je te dérange pourtant ? Non j'ai la politesse de t'ignorer. Laisse moi tu veux.~
 
 //Valygar  30xx  ///////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@3000  = ~Eloigne toi de moi, dragonne.~
-@3001  = ~Donne moi encore un ordre, et tu poura admirer un corps fumant d'acide de l'interieur. C'est tellement plaisant que les Bipedes ne peuvent s'empecher d'hurler de plaisir.~
-@3002  = ~Tu ne me ferra rien en presence de <CHARNAME>.~
-@3003  = ~Oh... une cervelle fonctionnelle ? Il me suffira d'attendre qu'il se lasse de toi pour t'apprendre le respect.~
-@3004  = ~Tu as une langue venimeuse, mais guere plus.~
+//N?1
+@3000  = ~Éloigne toi de moi, dragonne.~
+@3001  = ~Donne moi encore un ordre, et tu pourras admirer un corps fumant d'acide de l'intérieur. C'est tellement plaisant que les Bipèdes ne peuvent s'empêcher d'hurler de plaisir.~
+@3002  = ~Tu ne me feras rien en présence de <CHARNAME>.~
+@3003  = ~Oh... une cervelle fonctionnelle ? Il me suffira d'attendre qu'il se lasse de toi pour t'apprendre le respect.~ ~Oh... une cervelle fonctionnelle ? Il me suffira d'attendre qu'elle se lasse de toi pour t'apprendre le respect.~
+@3004  = ~Tu as une langue venimeuse, mais guère plus.~
 @3005  = ~Tu apprendras ton erreur un jour, mortel. Un jour...~
 
 //Anomen 31xx //////////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
+//N?1
 @3101   = ~ (Ricane) ~
 @3102   = ~Pourquoi ce rire quand tu me regardes Lysre.~
-@3103   = ~Je regarde un imbecile qui revere un Dieu protegeant les faibles. C'est un spectacle comique.~
-@3104   = ~Tu as tord de blasphemer! J-.~
-@3105   = ~Sssssh ! Garde tes menaces, je ne reconnais aucun de vos Dieux Bipedes. Viendra le jour ou Tiamat les devoreras. Vous retournez a votre place, rampant sur le sol comme des vers.~
-@3106   = ~Heaum voit Lysre!~
+@3103   = ~Je regarde un imbécile qui révère un Dieu protégeant les faibles. C'est un spectacle comique.~
+@3104   = ~Tu as tort de blasphémer ! J-.~
+@3105   = ~Ssssh ! Garde tes menaces, je ne reconnais aucun de tes Dieux Bipèdes. Viendra le jour où Tiamat les dévorera. Vous retournez à votre place, rampant sur le sol comme des vers.~
+@3106   = ~Heaume voit Lysre!~
 @3107   = ~Et je m'en moque.~
 
 //korgan 32xx ////////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@3200  = ~Lysre et Korgan se fond face.~
-@3201  = ~J'ai tue un mioche dans les bras de sa mere !~
-@3202  = ~J'ai manger une humain devant les yeux de sa portee !~
-@3203  = ~J'ai couper un type en deux, il courrait encore comme un cafard !~
+//N?1
+@3200  = ~Lysre et Korgan se font face.~
+@3201  = ~J'ai tué un mioche dans les bras de sa mère !~
+@3202  = ~J'ai mangé une humaine devant les yeux de sa portée !~
+@3203  = ~J'ai coupé un type en deux, il courrait encore comme un cafard !~
 @3204  = ~J'ai dissous le visage d'une gosse qui s'en pamait !~
 @3205  = ~*Regard mauvais*~
 @3206  = ~*Shhhhhhh!*~
 @3205  = ~*Rire* Je t'aime bien !~
-@3206  = ~Mpf, tu n'es pas insuportable.~
+@3206  = ~Mpf, tu n'es pas insupportable.~
 
 //Cernd   33XX  /////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
-@3300  = ~<CHARNAME> aime s'entourer de creatures inutiles~
-@3301  = ~Inutiles ? Je preserve l'equilibre Dragonne ! J-~
-@3302  = ~Les choses sont vouees a changer Bipedes. Petit a petit, Tiamat se prepare a boulverser l'odre etablit, et, louee soit elle. Je l'aiderai dans la mesure de mes capacites.~
+//N?1
+@3300  = ~<CHARNAME> aime s'entourer de créatures inutiles~
+@3301  = ~Inutiles ? Je préserve l'équilibre Dragonne ! J-~
+@3302  = ~Les choses sont vouées à changer Bipèdes. Petit à petit, Tiamat se prépare à bouleverser l'ordre établi, et, louée soit-elle. Je l'aiderai dans la mesure de mes capacités.~
 @3303  = ~Que peut faire un seul Dragon !~
-@3304  = ~Et que peut faire un seul Bipede.~
+@3304  = ~Et que peut faire un seul Bipède.~
 
 //Mazzy   34XX  /////////////////////////////////////////////////////////////////////////////////////////////////
-//N°1
+//N?1
 @3400  = ~*Toise Lysre*~
-@3401  = ~Je ne t'aime pas non plus, petite bipede. Ceux qui porte l'epee comme toi ont une sale tendance a nous deranger nous les dragons. Enfin, le probleme principal ce sont vos armures, une plaie a manger, et ca donne un arriere gout a la chair faisandee~
-@3402  = ~Et parfois on terrasse un dragon malefique.~
-@3403  = ~HAHAHA, si tu as pu tuer un dragon suffisament faible, satisfait toi de ta victoire. Les faibles n'ont pas leur place en ce monde.~
\ No newline at end of file
+@3401  = ~Je ne t'aime pas non plus, petite bipède. Ceux qui portent l'épée comme toi ont une sale tendance à nous déranger nous les dragons. Enfin, le problème principal ce sont vos armures, une plaie à manger, et ça donne un arrière goût à la chair faisandée~
+@3402  = ~Et parfois on terrasse un dragon maléfique.~
+@3403  = ~HAHAHA, si tu as pu tuer un dragon suffisamment faible, satisfais toi de ta victoire. Les faibles n'ont pas leur place en ce monde.~
