--- Lysre\French\BCKLys25.tra
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Lysre\\French\\BCKLys25.tra"	
@@ -3,24 +3,24 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 //N°1
-@0    = ~<CHARNAME> ! Sachez que je suis tout a fait ravie d'avoir regler ma part du contrat! Bientot, vous devriez pouvoir faire de même...~
-@1    = ~Comment ca, tu as fait ta part ?~
+@0    = ~<CHARNAME> ! Sachez que je suis tout à fait ravie d'avoir réglé ma part du contrat! Bientôt, vous devriez pouvoir faire de même...~
+@1    = ~Comment ça, tu as fait ta part ?~
 @2    = ~Et je te remercie de ton aide.~
 @3    = ~Ah, laisse moi tranquille Lysre!~
 @4    = ~*Ne rien dire*~
-@5    = ~La mort d'Irenicus et le retour de votre ame sont insuffisante pour vous bipedes? Decidement, vous êtes detestable.~
-@6    = ~Je l'ai fait dans mon interet <CHARNAME>. Même si je reconnais y avoir pris plaisir.~
-@7    = ~Ah... vous êtes ennuyant, vivement que nos routes se separent.~
-@8    = ~*Lysre vous toise, apparament agacee par l'absence de reaction de la part de <CHARNAME>.~
+@5    = ~La mort d'Irenicus et le retour de votre âme sont insuffisants pour vous bipèdes? Décidément, vous êtes détestable.~
+@6    = ~Je l'ai fait dans mon intérêt <CHARNAME>. Même si je reconnais y avoir pris plaisir.~
+@7    = ~Ah... vous êtes ennuyant, vivement que nos routes se séparent.~
+@8    = ~*Lysre vous toise, apparemment agacée par l'absence de réaction de la part de <CHARNAME>.~
 
 //N°2
-@23   = ~Une lutte fratricide... On doit reconnaitre a Bhaal un certain gout pour l'education de sa progeniture, vous n'etes pas d'accord <CHARNAME>?~
-@24   = ~Tu trouve ca "normale" lysre ?~
-@25   = ~C'est ma destinee que ca me plaise ou non.~
-@26   = ~J'adore ca... Tuer les faibles, laisser leur corps dans mon sillage.~
-@27   = ~Et bien oui, quand j'aurais trouver des males dignes de s'occuper de mes oeufs, ils auront un role similaire pour s'assurer que le meilleur subsiste.~
-@28   = ~*Lysre vois toise avec dedain* La destinee... Un jour Tiamat reignera, c'est la seule destinee qui merite mon attention.~
-@29   = ~Voila des propos plein d'inteligence <CHARNAME>! Poursuivons cette merveilleuse lancee !~
+@23   = ~Une lutte fratricide... On doit reconnaître à Bhaal un certain goût pour l'éducation de sa progéniture, vous n'êtes pas d'accord <CHARNAME>?~
+@24   = ~Tu trouves ça "normale" Lysre ?~
+@25   = ~C'est ma destinée que ça me plaise ou non.~
+@26   = ~J'adore ça... Tuer les faibles, laisser leur corps dans mon sillage.~
+@27   = ~Et bien oui, quand j'aurais trouvé des mâles dignes de s'occuper de mes oeufs, ils auront un rôle similaire pour s'assurer que le meilleur subsiste.~
+@28   = ~*Lysre vous toise avec dédain* La destinée... Un jour Tiamat régnera, c'est la seule destinée qui mérite mon attention.~
+@29   = ~Voilà des propos plein d'intelligence <CHARNAME>! Poursuivons cette merveilleuse lancée !~
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //Banters avec les NPCs  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
@@ -29,10 +29,10 @@
 @45   = ~Lysre ! Laisse moi !~
 @46   = ~Je te fais honneur de mon regard. Tu devrais me remercier.~
 @47   = ~Tu parles, je sais que tu te moques de moi !~
-@48   = ~Je me moque de chaque bipede, mais je dois avouer que l'oiseau sans ailes est des plus pitoyables oui.~
+@48   = ~Je me moque de chaque bipède, mais je dois avouer que l'oiseau sans ailes est des plus pitoyables oui.~
 @49   = ~Eh bien, je n'ai pas besoin de ton attention ! Laisse moi tranquille, je t'en prie !~
-@50   = ~Sinon, tu vais pleurer dans les juppons de <CHARNAME>? (voix de fausset) <CHARNAME> <CHARNAME> ! Tchip Tchip Tchip !~
-@51   = ~*S'eloigne en fulminant.*~
+@50   = ~Sinon, tu vas pleurer dans les jupons de <CHARNAME>? (voix de fausset) <CHARNAME> <CHARNAME> ! Tchip Tchip Tchip !~
+@51   = ~*S'éloigne en fulminant.*~
 @52   = ~*Ricane*~
 
 //Anomen
@@ -59,20 +59,20 @@
 @69   = ~Rien~
 
 //Imoen
-@73   = ~Je ne te comprend pas femelle... Pourquoi te contenter de ta place.~
+@73   = ~Je ne te comprends pas femelle... Pourquoi te contenter de ta place.~
 @74   = ~Mais qu'est ce que tu me raconte encore Lysre?~
-@75   = ~Tu es une femelle, d'essence divine. Je reconnais a <CHARNAME> beaucoup de qualites, pour un bipede, mais toi... Tu vis dans l'ombre.~
+@75   = ~Tu es une femelle, d'essence divine. Je reconnais à <CHARNAME> beaucoup de qualités, pour un bipède, mais toi... Tu vis dans l'ombre.~
 @76   = ~Je-~
-@77   = ~*S'eloigne d'un coup d'aile*~
-@78   = ~Elle... s'inquiete pour moi ou elle se moque ?~
+@77   = ~*S'éloigne d'un coup d'aile*~
+@78   = ~Elle... s'inquiète pour moi ou elle se moque ?~
 
 //Jaheira
-@79   = ~Je me demande pourquoi <CHARNAME> te garde a ses cotes.~
-@80   = ~C'est sur que toi l'insuportable moralisatrice, tu es un meilleur choix.~
-@81   = ~Le concept meme de morale t'es inconnu Lysre !*~
-@82   = ~Oh, je sais ce que c'est, la preuve de la betise. Tu es un exemple flagrant.~
+@79   = ~Je me demande pourquoi <CHARNAME> te garde à ses côtés.~
+@80   = ~C'est sur que toi l'insupportable moralisatrice, tu es un meilleur choix.~
+@81   = ~Le concept même de morale t'es inconnu Lysre !*~
+@82   = ~Oh, je sais ce que c'est, la preuve de la bêtise. Tu es un exemple flagrant.~
 @83   = ~Qu-~
-@84   = ~Shhh, nous participons a l'e-qui-li-bre aux cotes de <CHARNAME> tu ne voudrais pas te contredire ? Enfin, ca ne serait pas la premiere fois.~
+@84   = ~Shhh, nous participons à l'é-qui-libre aux côtés de <CHARNAME> tu ne voudrais pas te contredire ? Enfin, ça ne serait pas la première fois.~
 @85   = ~Inutile de me baver tes salades. Ton heure viendra rapidement, fais moi confiance.~
 
 //Jan
@@ -81,56 +81,56 @@
 @90   = ~Laisse moi tranquille !    ~
 
 //Korgan
-@99   = ~On s'ennuit... <CHARNAME>! Allons plonger nos lames dans le sang !~
+@99   = ~On s'ennuie... <CHARNAME>! Allons plonger nos lames dans le sang !~
 @100  = ~Oui, allons donc massacrer une famille !~
-@101  = ~Bonne idee, nous commencerons par les enfants !~
-@102  = ~Ne devrions nous pas les laisser choisir qui mourra le premier? Histoire de tester les liens familliaux?~
+@101  = ~Bonne idée, nous commencerons par les enfants !~
+@102  = ~Ne devrions nous pas les laisser choisir qui mourra le premier? Histoire de tester les liens familiaux ?~
 @103  = ~<CHARNAME>, ma hache me démange maintenant !~
 
 //Mazzy
-@104  = ~Toujours la toi...~
-@105  = ~L'idee te derange dragonne?~
-@106  = ~Oui, tu es une menace pour les plus miserables de mes congeneres. Etrangement, cette idee m'agace aujourd'hui.~
+@104  = ~Toujours là toi...~
+@105  = ~L'idée te dérange dragonne?~
+@106  = ~Oui, tu es une menace pour les plus misérables de mes congénères. Étrangement, cette idée m'agace aujourd'hui.~
 @107  = ~Incroyable, tu fais preuve de sympathie.~
-@108  = ~Laisse moi tranquille, je m'en voudrai de te tuer. <CHARNAME> n'apprecierait pas.~
-@109  = ~Reglons nos comptes une fois cette histoire terminee.~
+@108  = ~Laisse moi tranquille, je m'en voudrais de te tuer. <CHARNAME> n'apprécierait pas.~
+@109  = ~Réglons nos comptes une fois cette histoire terminée.~
 @110  = ~Vivement ce jour.~
 
 //Minsc
 @112  = ~(Squiik !)~
 @113  = ~Exactement, mais il faut le supporter. Ils ne peuvent comprendre.~
 @115  = ~(Squiik !)~
-@117  = ~Non, je crois que tu le surestime.~
+@117  = ~Non, je crois que tu le surestimes.~
 @118  = ~(Squiik !)~
 @119  = ~Ne tente pas ta chance, il me suffit d'un instant pour t'avaler...~
 
 //Nalia
-@120  = ~Tu es desormais une grande mage petite... Mais je sais que tu restes aussi sotte.~
-@121  = ~Nos aventures m'ont apprise beaucoup de chose. C'est le devoir des gens puissants de proteger les plus demunis pour qu'ils profitent de leurs existences. C'est un honneur de les servir.~
-@122  = ~Tu vas passer une vie minable, a nourrir les sangsues qui te colleront, quel decheance pour une femelle puissante.~
-@123  = ~Vous serez eternellement incapable de me comprendre Lysre.~
+@120  = ~Tu es désormais une grande mage petite... Mais je sais que tu restes aussi sotte.~
+@121  = ~Nos aventures m'ont appris beaucoup de choses. C'est le devoir des gens puissants de protéger les plus démunis pour qu'ils profitent de leurs existences. C'est un honneur de les servir.~
+@122  = ~Tu vas passer une vie minable, à nourrir les sangsues qui te colleront, quel déchéance pour une femelle puissante.~
+@123  = ~Vous serez éternellement incapable de me comprendre Lysre.~
 
 //Sarevok
-@124  = ~On dirait que mon frere s'entoure d'etrange compagnons~ ~On dirait que ma soeur s'entoure d'etrange compagnons~
-@125  = ~Il est vrai que choisir un ancien ennemi qu'on a soit meme tuer est un choix que beaucoup jugerait stupide.~
-@126  = ~<CHARNAME> a besoin de puissance pour parvenir a ses fins, c'etait un choix logique.~
-@127  = ~La encore, je trouve que pour un revenant, tu es exceptionnel... J'aurais peut etre prefere te voir a la place de <CHARNAME> Je gage que ta compagnie aurait ete des plus... amusantes...~
-@128  = ~Comme si j'aurais eu la betise de te faire confiance, tu es aussi venimeuse que ta salive.~
-@129  = ~*Ricane en s'eloignant*~
+@124  = ~On dirait que mon frère s'entoure d'étranges compagnons~ ~On dirait que ma soeur s'entoure d'étranges compagnons~
+@125  = ~Il est vrai que choisir un ancien ennemi qu'on a soit même tuer est un choix que beaucoup jugerait stupide.~
+@126  = ~<CHARNAME> a besoin de puissance pour parvenir à ses fins, c'était un choix logique.~
+@127  = ~Là encore, je trouve que pour un revenant, tu es exceptionnel... J'aurais peut-être préféré te voir à la place de <CHARNAME>. Je gage que ta compagnie aurait été des plus... amusantes...~
+@128  = ~Comme si j'avais eu la bêtise de te faire confiance, tu es aussi venimeuse que ta salive.~
+@129  = ~*Ricane en s'éloignant*~
 
 //Valygar
 @133  = ~Lysre, quels sont tes projets une fois loin de <CHARNAME>?~
-@134  = ~Rien de suprenant, etendre l'influence de Tiamat, pour devenir une Ravageuse... Aaah, vivement les hurlements des foules sous mes pluies d'acides.~
-@135  = ~Compter sur moi pour t'en empecher. Je vais t'arreter tant que tu es encore faible.~
+@134  = ~Rien de surprenant, étendre l'influence de Tiamat, pour devenir une Ravageuse... Aaah, vivement les hurlements des foules sous mes pluies d'acide.~
+@135  = ~Compter sur moi pour t'en empêcher. Je vais t'arrêter tant que tu es encore faible.~
 @136  = ~He, Valygar, calmez-vous ! Lysre est notre compagnon, souvenez-vous !~
-@137  = ~Elle cherche juste a vous poussez a bout, ignorez la.~
+@137  = ~Elle cherche juste à vous poussez à bout, ignorez là.~
 @138  = ~*Les ignorer*~
-@139  = ~Comme c'est touchant <CHARNAME> Je vais finir par croire que vous m'appreciez a ma juste valeur.~
-@140  = ~Je ne verserai pas ton sang aujourd'hui, nous avons une quete a accomplir. Mais je te tuerai dragonne.~
-@150  = ~Je t'attendrai, te tuerais, et tu pourriras sous ton cabanon je te le promet.~
+@139  = ~Comme c'est touchant <CHARNAME> Je vais finir par croire que vous m'appreciez à ma juste valeur.~
+@140  = ~Je ne verserai pas ton sang aujourd'hui, nous avons une quête à accomplir. Mais je te tuerai dragonne.~
+@150  = ~Je t'attendrai, te tuerai, et tu pourriras sous ton cabanon je te le promets.~
 
 //Viconia
-@141  = ~Lysre, pourquoi me fixe tu ?~
-@142  = ~Je regrettais ta fidelite a Shar. Une Drow aussi talentueusse que toi aurait ete une si grande pretresse pour Tiamat.~
-@143  = ~Nulle Drow ne se soumettrais a Tiamat ! la reine des dragon ne reconnait que ses propres enfants.~
-@144  = ~Ton intelligence et ta culture t'honore, femelle, ca n'accentue que d'avantage ma deception.~
\ No newline at end of file
+@141  = ~Lysre, pourquoi me fixe-tu ?~
+@142  = ~Je regrette ta fidélité à Shar. Une Drow aussi talentueuse que toi aurait été une si grande prêtresse pour Tiamat.~
+@143  = ~Nulle Drow ne se soumettrait à Tiamat ! La reine des dragons ne reconnait que ses propres enfants.~
+@144  = ~Ton intelligence et ta culture t'honore, femelle, ça n'accentue que davantage ma déception.~
