--- Lysre\Dialogue\Quete1\CKKobold.d
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Lysre\\Dialogue\\Quete1\\CKKobold.d"	
@@ -4,10 +4,10 @@
 BEGIN CKKobolds
 
 IF ~True()~ THEN BEGIN CKKob1
-  SAY ~*Ce Kobold vous observe avec curiosite* Grrrr Graaaa Rrrrou~
+  SAY ~*Ce Kobold vous observe avec curiosité* Grrrr Graaaa Rrrrou~
   IF ~~ THEN REPLY ~Que fais-tu ici?~ GOTO CKKob2
   IF ~~ THEN REPLY ~Qui es tu ?~ GOTO CKKob3
-  IF ~~ THEN REPLY ~*S'eloigner*~ EXIT
+  IF ~~ THEN REPLY ~*S'éloigner*~ EXIT
 END
 
 IF ~~ THEN BEGIN CKKob2
@@ -19,6 +19,3 @@ IF ~~ THEN BEGIN CKKob3
   SAY ~Rrr rrr Kay kay ! Rrrr !.~
   IF ~~ THEN EXIT
 END
-
-
-
