--- Lysre\Dialogue\Quete1\CKKobol2.d
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Lysre\\Dialogue\\Quete1\\CKKobol2.d"	
@@ -1,5 +1,5 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// Dialogue des fideles de Tiamat ////////////////////////////////////////////////////////////////////////////////////////
+// Dialogue des fidèles de Tiamat ////////////////////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 BEGIN CKKoboldalpha
 
@@ -7,18 +7,15 @@ IF ~True()~ THEN BEGIN CKKoba1
   SAY ~*Ce kobold vous observe calmement*~
   IF ~~ THEN REPLY ~Que fais-tu ici?~ GOTO CKKoba2
   IF ~~ THEN REPLY ~Qui es tu ?~ GOTO CKKoba3
-  IF ~~ THEN REPLY ~*S'eloigner*~ EXIT
+  IF ~~ THEN REPLY ~*S'éloigner*~ EXIT
 END
 
 IF ~~ THEN BEGIN CKKoba2
-  SAY ~*Le kobold hoche lentement la tete en direction de la fosse*~
+  SAY ~*Le kobold hoche lentement la tête en direction de la fosse*~
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN CKFid3
-  SAY ~*Le kobold secoue tristement la tete*~
+  SAY ~*Le kobold secoue tristement la tête*~
   IF ~~ THEN EXIT
 END
-
-
-
