--- Lysre\Dialogue\CKLysreP.d
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Lysre\\Dialogue\\CKLysreP.d"	
@@ -2,21 +2,21 @@
 
 BEGIN CKLysreP
 
-// Rupture de contrat a cause du comportement du joueur. ///////////////////////////////////////////////////////////////////////////////////////////////////
+// Rupture de contrat à cause du comportement du joueur. ///////////////////////////////////////////////////////////////////////////////////////////////////
 IF ~Global("CKLysreKickedOut","GLOBAL",0) HappinessLT(Myself,-299)~ CKLysreVeryEvilKickedOut
-SAY ~J'en ai assez de vous, de vos manieres. Puissiez vous vous etouffer dans votre sommeil et finir au plus profond des enfers.~
+SAY ~J'en ai assez de vous, de vos manières. Puissiez vous vous étouffer dans votre sommeil et finir au plus profond des enfers.~
 IF ~~ DO ~EscapeArea()~ EXIT
 END
 
-//Rupture de contrat a cause de deux abandons //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//Rupture de contrat à cause de deux abandons //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 IF ~Global("CKLysreKickedOut","GLOBAL",2) HappinessLT(Myself,-299)~ CKLysreVeryEvilKickedOut2
-SAY ~Je vous avez prevenu <CHARNAME> Je ne suis pas un chien vous suivant selon votre bon vouloir. Vous vendez de rompre notre contrat. J'en ai assez de vous, de vos manieres. Puissiez vous vous etouffer dans votre sommeil et finir au plus profond des enfers.~
+SAY ~Je vous avais prévenu <CHARNAME> Je ne suis pas un chien vous suivant selon votre bon vouloir. Vous vendez de rompre notre contrat. J'en ai assez de vous, de vos manières. Puissiez vous vous étouffer dans votre sommeil et finir au plus profond des enfers.~
 IF ~~ DO ~EscapeArea()~ EXIT
 END
 
-// Se debarasser du NPC  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Se débarrasser du NPC  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 IF ~Global("CKLysreKickedOut","GLOBAL",0)~ CKLysreKickedOut1
-SAY ~Un instant ? J'ai du mal comprendre, vous me demandez de partir?! Vous devez rester pres de moi, vous vous en souvenez?~
+SAY ~Un instant ? J'ai du mal comprendre, vous me demandez de partir?! Vous devez rester près de moi, vous vous en souvenez?~
 ++ ~Mais pas du tout Lysre. Reprenons notre route.~ DO ~JoinParty()~ EXIT
 + ~!AreaCheck("AR0603")
 !AreaCheck("AR0410")
@@ -37,7 +37,7 @@ SAY ~Un instant ? J'ai du mal comprendre, vous me demandez de partir?! Vous deve
 !AreaCheck("AR1506")
 !AreaCheck("AR1507")
 !AreaCheck("AR1508")
-!AreaCheck("AR0710")~ + ~Nos routes doivent se separer un moment, retrouve moi a la promenade de Walkyne.~ DO ~SetGlobal("CKLysreKickedOut","GLOBAL",1) EscapeArea() MoveGlobal("AR710","CKLysre",[481.182])~ EXIT
+!AreaCheck("AR0710")~ + ~Nos routes doivent se séparer un moment, retrouvez moi à la promenade de Walkyne.~ DO ~SetGlobal("CKLysreKickedOut","GLOBAL",1) EscapeArea() MoveGlobal("AR710","CKLysre",[481.182])~ EXIT
 + ~!AreaCheck("AR0301")
 !AreaCheck("AR0302")
 !AreaCheck("AR0303")
@@ -51,23 +51,23 @@ SAY ~Un instant ? J'ai du mal comprendre, vous me demandez de partir?! Vous deve
 !AreaCheck("AR1102")
 !AreaCheck("AR1107")
 !AreaCheck("AR0801")
-!AreaCheck("AR0803")~ + ~Ecoute, j'ai "besoin" que tu m'attendes ici un moment.~ DO ~SetGlobal("CKLysreKickedOut","GLOBAL",1)~ EXIT
+!AreaCheck("AR0803")~ + ~Écoute, j'ai "besoin" que tu m'attendes ici un moment.~ DO ~SetGlobal("CKLysreKickedOut","GLOBAL",1)~ EXIT
 END
 
 // Revenir chercher Lysre  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 IF ~Global("CKLysreKickedOut","GLOBAL",1)~ CKLysreMetAfterKickOut1
-SAY ~Enfin vous revoila, qu'attendiez vous pour venir me chercher ?~
+SAY ~Enfin vous revoilà, qu'attendiez vous pour venir me chercher ?~
 ++ ~J'ai eu un petit contretemps, reprenons notre chemin ensemble Lysre.~ + CKLysreMetAgain1
 ++ ~J'ai besoin d'encore un peu de temps. Desole.~ + CKLysreVeryEvilKickedOut3
 END
 
 IF ~~ CKLysreMetAgain1
-SAY ~Bien, mais notez ceci <CHARNAME> c'est la seule et unique derniere fois que je tolere vos caprices, <CHARNAME>~
+SAY ~Bien, mais notez ceci <CHARNAME> c'est la seule et unique dernière fois que je tolère vos caprices, <CHARNAME>~
 IF ~~ DO ~SetGlobal("CKLysreKickedOut","GLOBAL",2) JoinParty()~ EXIT
 END
 
 IF ~Global("CKLysreKickedOut","GLOBAL",2) HappinessLT(Myself,-299)~ CKLysreVeryEvilKickedOut3
-SAY ~Je vous avez prevenu <CHARNAME> Je ne suis pas un chien vous suivant selon votre bon vouloir. Vous voulez rompre notre contrat ? ET BIEN D'ACCORD ! J'en ai assez de vous, de vos manieres. Puissiez vous vous etouffer dans votre sommeil et finir au plus profond des enfers.~
+SAY ~Je vous avais prévenu <CHARNAME> Je ne suis pas un chien vous suivant selon votre bon vouloir. Vous voulez rompre notre contrat ? ET BIEN D'ACCORD ! J'en ai assez de vous, de vos manières. Puissiez vous vous étouffer dans votre sommeil et finir au plus profond des enfers.~
 IF ~~ DO ~EscapeArea()~ EXIT
 END
