--- ua\ua.tp2	
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetup\\OverwriteFiles\\BWP\\BiG World Fixpack\\ua\\ua.tp2"	
@@ -13,6 +13,7 @@ END
 
 LANGUAGE ~English~ ~english~ ~%MOD_FOLDER%/tra/english/game.tra~ ~%MOD_FOLDER%/tra/english/setup.tra~
 LANGUAGE ~Russian~ ~russian~ ~%MOD_FOLDER%/tra/russian/game.tra~ ~%MOD_FOLDER%/tra/russian/setup.tra~
+LANGUAGE ~French~ ~french~ ~ua/tra/french/game.tra~ ~ua/tra/french/setup.tra~
 
 
 BEGIN @1 //+1
