--- Hanna\Hanna.tp2
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\Hanna\\Hanna.tp2"	
@@ -5,7 +5,7 @@ AUTHOR ~drizzt905@hotmail.com~
 // There are several changes to use new features of WeiDU
 
 // Version displayed in the WeiDU.log
-VERSION ~v2.5~
+VERSION ~v2.5 BWS-FR Fix~
 
 README ~Hanna/Readme_Hanna.doc~
 
@@ -291,8 +291,8 @@ COMPILE ~HANNA/BAF/estrol0.baf~
 
 
 COMPILE ~HANNA/BAF/zru4.baf~
-APPEND ~ANIMATE.IDS~
-~0xE060 IC_LICH~
+APPEND ~animate.ids~ ~0xE060 IC_LICH~ UNLESS ~^0xE060~
+
 COPY ~HANNA/bam/MLICA1.BAM~ ~override/MLICA1.BAM~
 COPY ~HANNA/bam/MLICA1E.BAM~ ~override/MLICA1E.BAM~
 COPY ~HANNA/bam/MLICA2.BAM~ ~override/MLICA2.BAM~
@@ -393,7 +393,7 @@ COPY ~HANNA/CRE/HAENE06.CRE~ ~override/HAENE06.CRE~
    SAY NAME2 @95
 
 
-//0==[[=Misión=>
+//0==[[=Misiï¿½n=>
 
 COPY ~HANNA/item/HANDAG0.itm~ ~override/HANDAG0.itm~
 SAY NAME1 @66
@@ -527,8 +527,8 @@ BEGIN @134
 EXTEND_TOP ~ar2013.bcs~ ~HANNA/baf/HAESQ02.baf~
 COMPILE ~HANNA/BAF/HAESQ01.baf~
 
-APPEND ~ANIMATE.IDS~
-~0xE060 IC_LICH~
+APPEND ~animate.ids~ ~0xE060 IC_LICH~ UNLESS ~^0xE060~
+
 COPY ~HANNA/bam/MLICA1.BAM~ ~override/MLICA1.BAM~
 COPY ~HANNA/bam/MLICA1E.BAM~ ~override/MLICA1E.BAM~
 COPY ~HANNA/bam/MLICA2.BAM~ ~override/MLICA2.BAM~
