--- LR\dlg\finaleA.d
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetup\\OverwriteFiles\\Patchs\\LR\\dlg\\finaleA.d"	
@@ -3,11 +3,22 @@
 //And as things stand now, the LR version will cause some of the Ascension-Jon dialogue to not be shown.
 //Much of the code is more or less stolen from the Redemption Mod.
 REPLACE Irenic2
-IF ~Global("FinalFight","AR6200",2)~ 1
+IF WEIGHT #-2~Global("FinalFight","AR6200",2)~THEN BEGIN LRReplace1
   SAY @0
   ++ @1 + lrAscJon1
   ++ @2 + lrAscJon1
 END
+
+IF ~~ lrAscJon1
+  SAY @80
+  ++ @81 + lrAscJon2
+  ++ @82 + lrAscJon2
+END
+
+IF ~~ lrAscJon2
+  SAY @83 
+  IF ~~ + 7
+END
 END
 
 BEGIN LRGODEMO
@@ -232,11 +243,29 @@ IF ~~ lrdemo11
       DO ~ClearAllActions() StartCutSceneMode() StartCutScene("redgdem3")~ EXIT
 END
 
+APPEND JONEL25J
+IF ~~ jondemo1
+  SAY @93
+  ++ @94 EXTERN lrgodemo lrdemo4
+  ++ @95 EXTERN lrgodemo lrdemo4
+  ++ @96 EXTERN lrgodemo lrdemo4
+  ++ @97 + jondemo2
+END
+
+IF ~~ jondemo2
+  SAY @98
+  ++ @99 EXTERN lrgodemo lrdemo4
+END
+
+IF ~~ jondemo3
+  SAY @100
+  IF ~~ THEN EXTERN LRGODEMO lrdemo9
+END
+END
+
 
 ADD_TRANS_TRIGGER IMOEN25P 4 ~!InParty("lrirenic") G("lrJonToFinal",0) G("AcceptIlmater",0)~
 
-ADD_TRANS_TRIGGER FINSOL01 38 ~False()~ DO 1
-
 EXTEND_BOTTOM FINBODH 0
 IF ~InParty("lrirenic")~ EXTERN JONEL25J lrBodhiJonMelissan
 END
@@ -264,19 +293,6 @@ IF ~OR(2) InParty("lrIrenic") G("lrJonToFinal",1) OR(2) !Dead("lrirenic") G("Acc
 IF ~Dead("lrirenic") OR(2) G("lrHasSoulStone",1) !G("AcceptIlmater",0) Global("lrFinSolJon","LOCALS",0)~ DO ~SetGlobal("lrFinSolJon","LOCALS",1)~ EXTERN FINSOL01 5
 END
 
-APPEND Irenic2
-IF ~~ lrAscJon1
-  SAY @80
-  ++ @81 + lrAscJon2
-  ++ @82 + lrAscJon2
-END
-
-IF ~~ lrAscJon2
-  SAY @83 
-  IF ~~ + 7
-END
-END
-
 APPEND FINMEL01
 /*IF ~~ MelA
   SAY @120
@@ -327,26 +343,6 @@ IF ~~ Bodhi2
 END
 END
 
-APPEND JONEL25J
-IF ~~ jondemo1
-  SAY @93
-  ++ @94 EXTERN lrgodemo lrdemo4
-  ++ @95 EXTERN lrgodemo lrdemo4
-  ++ @96 EXTERN lrgodemo lrdemo4
-  ++ @97 + jondemo2
-END
-
-IF ~~ jondemo2
-  SAY @98
-  ++ @99 EXTERN lrgodemo lrdemo4
-END
-
-IF ~~ jondemo3
-  SAY @100
-  IF ~~ THEN EXTERN LRGODEMO lrdemo9
-END
-END
-
 CHAIN JONEL25J lrBodhiJonMelissan
    @101
   == FINMEL01
