--- setup-Sheena.TP2	Mon Jan 16 14:32:13 2017
+++ C:\BWP Patchstudio\patched files\setup-Sheena.TP2	Fri Mar 02 11:07:19 2018
@@ -1,7 +1,7 @@
 BACKUP ~Sheena\backup~
 AUTHOR ~edvin@seznam.cz~
 
-VERSION ~v2.6~
+VERSION ~v2.6 BWP Fix~
 
 AUTO_TRA ~Sheena/%s~
 
@@ -75,8 +75,8 @@
 END
 
 APPEND ~Action.ids~
-~147 RemoveSpellRES(S:RES*,O:Target)~
-UNLESS ~147 RemoveSpellRES(S:RES*,O:Target)~
+~147 RemoveSpellRES(S:Spell*)~
+UNLESS ~147 RemoveSpellRES(S:Spell*)~
 APPEND ~Trigger.ids~
 ~0x4031 HaveSpellRES(S:RES*)~
 UNLESS ~0x4031 HaveSpellRES(S:RES*)~
