--- bg2improvedgui\bg2improvedgui.tp2
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\bg2improvedgui\\bg2improvedgui.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~bg2improvedgui/backup~
 AUTHOR ~Insomniator (http://www.shsforums.net/topic/60986-bg2-impoved-gui/)~
 
-VERSION ~3.18~
+VERSION ~3.19 BWP-FR fix~
 
 LANGUAGE ~English~ ~english~
          ~bg2improvedgui/tra/english/setup.tra~
@@ -12,6 +12,9 @@ LANGUAGE ~
 LANGUAGE ~Русский~ ~russian_1251~
          ~bg2improvedgui/tra/russian_1251/setup.tra~
          ~bg2improvedgui/tra/russian_1251/strings.tra~
+LANGUAGE ~French~ ~french~
+         ~bg2improvedgui/tra/french/setup.tra~
+         ~bg2improvedgui/tra/french/strings.tra~
 
 ////////////////////////////////
 //  Loot panel
