--- stratagems\abazigal\abazigal_enclave.tpa	
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetup\\OverwriteFiles\\BWP\\stratagems\\abazigal\\abazigal_enclave.tpa"	
@@ -809,12 +809,12 @@ COPY_EXISTING ar6008.are override
 COPY_EXISTING 01pool4.bcs override
    DECOMPILE_AND_PATCH BEGIN
       REPLACE_TEXTUALLY ~ActionOverride(Player1,LeaveAreaLUAPanicEntry([^)]*))~ ~~
-      REPLACE_TEXTUALLY "Player1,LeaveAreaLUAEntry([^)]*)" ~Player1,LeaveAreaLUA("AR6008","",[637.384],S)~
-      REPLACE_TEXTUALLY "Player2,LeaveAreaLUAEntry([^)]*)" ~Player2,LeaveAreaLUA("AR6008","",[683.378],S)~
-      REPLACE_TEXTUALLY "Player3,LeaveAreaLUAEntry([^)]*)" ~Player3,LeaveAreaLUA("AR6008","",[607.364],S)~
-      REPLACE_TEXTUALLY "Player4,LeaveAreaLUAEntry([^)]*)" ~Player4,LeaveAreaLUA("AR6008","",[725.358],S)~
-      REPLACE_TEXTUALLY "Player5,LeaveAreaLUAEntry([^)]*)" ~Player5,LeaveAreaLUA("AR6008","",[586.325],S)~
-      REPLACE_TEXTUALLY "Player6,LeaveAreaLUAEntry([^)]*)" ~Player6,LeaveAreaLUA("AR6008","",[755.304],S)~
+      REPLACE_TEXTUALLY "Player1,LeaveAreaLUAEntry([^)]*)" ~Player1,LeaveAreaLUA("AR6008","",[637.384],0)~
+      REPLACE_TEXTUALLY "Player2,LeaveAreaLUAEntry([^)]*)" ~Player2,LeaveAreaLUA("AR6008","",[683.378],0)~
+      REPLACE_TEXTUALLY "Player3,LeaveAreaLUAEntry([^)]*)" ~Player3,LeaveAreaLUA("AR6008","",[607.364],0)~
+      REPLACE_TEXTUALLY "Player4,LeaveAreaLUAEntry([^)]*)" ~Player4,LeaveAreaLUA("AR6008","",[725.358],0)~
+      REPLACE_TEXTUALLY "Player5,LeaveAreaLUAEntry([^)]*)" ~Player5,LeaveAreaLUA("AR6008","",[586.325],0)~
+      REPLACE_TEXTUALLY "Player6,LeaveAreaLUAEntry([^)]*)" ~Player6,LeaveAreaLUA("AR6008","",[755.304],0)~
    END
 BUT_ONLY
 
