--- stratagems\lang\french\leveller.tra
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\stratagems\\lang\\french\\leveller.tra"	
@@ -15,14 +15,14 @@
 @15 = ~Attribuer les talents de voleur~
 @16 = ~Ce personnage dispose de points de talent de voleur non dépensés, qui peuvent être dépensés à partir de la barre de capacités spéciales.~
 @17 = ~Potion d'expérience~
-@18 = ~Cette potion contient la sagesse des âges distillée. Quiconque la boit acquiert suffisamment d'expérience pour passer au niveau suivant. 
+@18 = ~Cette potion contient la sagesse des âges distillée. Quiconque la boit acquiert suffisamment d'expérience pour passer au niveau suivant.
 
 Les personnages multi-classés acquièrent suffisamment d'expérience pour atteindre le niveau suivant dans une classe, dans cet ordre de priorité : Voleur>Guerrier=Rôdeur>Clerc=Druide>Mage.~
 @19 = ~Les kits suivants sont disponibles :~
 @20 = ~Classe pure (sans kit)~
 @21 = ~Vous pouvez utiliser ce menu pour sélectionner la classe et le kit du personnage.
 
-Cette fonction ne tente, en aucun cas, de respecter les restrictions de race/classe, les scores de capacité minimums ou les exigences d'alignement. Ce principe se base ici sur l'honneur !
+Cette fonction ne tente, en aucun cas, de respecter les scores de capacité minimums ou les exigences d'alignement. Ce principe se base ici sur l'honneur ! Les restrictions de race/classe sont actuellement appliquées, mais vous pouvez les désactiver si vous le souhaitez.
 
 REMARQUE : si vous jouez avec l'IA de groupe désactivée, activez-la brièvement avant de monter votre personnage de niveau, si vous changez de classe.~
 @22 = ~Quitter sans faire d'autres changements.~
@@ -50,3 +50,14 @@ Ce pouvoir ne peut qu'
 @43 = ~Ce pouvoir unique augmente de façon permanente votre talent Détection de pièges de 5 %.~
 @44 = ~Ce pouvoir unique augmente de façon permanente votre talent Vol à la tire de 5 %.~
 @45 = ~Pour appliquer les sorts d'entropies spécifiques du personnage, sauvegardez et rechargez le jeu.~
+
+@46 = ~Tout compte fait, ne faites aucun changement pour l'instant.~
+@47 = ~Cesser d'appliquer les restrictions de race/classe et me permettre de choisir des classes illégales.~
+@48 = ~Rétablir les restrictions de race/classe et empêcher que des classes illégales ne puissent être sélectionnées.~
+@49 = ~Vous pouvez utiliser ce menu pour sélectionner la classe et le kit du personnage.
+
+Cette fonction ne tente, en aucun cas, de respecter les scores de capacité minimums ou les exigences d'alignement. Ce principe se base ici sur l'honneur ! Les restrictions de race/classe ne sont actuellement pas appliquées, mais vous pouvez les réactiver si vous le souhaitez.
+
+REMARQUE : si vous jouez avec l'IA de groupe désactivée, activez-la brièvement avant de monter votre personnage de niveau, si vous changez de classe.~
+@50 = ~Suppression des restrictions de classe/race. Ne laissez personne vous empêcher de poursuivre votre rêve !~
+@51 = ~Les restrictions de classe/race sont rétablies. Bon retour sur AD&D, vous nous avez manqué.~
