--- stratagems\lang\french\setup.tra
+++ "b/E:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFR\\OverwriteFiles\\BWP\\stratagems\\lang\\french\\setup.tra"	
@@ -216,3 +216,4 @@
 @50007 = ~Ce composant n'est desormais plus supporte (utilisez plutot le composant equivalent du mod Tweaks Anthology de Gibberlings 3 a la place)~
 @50008 = ~Ce composant est uniquement destine a Icewind Dale: Enhanced Edition~
 @50009 = ~Ce composant n'est pas compatible avec Icewind Dale: Enhanced Edition~
+@50010 = ~Ce composant n'est pas compatible avec le composant de Tweaks Anthology qui empeche les humains d'avoir acces a des classes jumelees~
