--- AC_QUEST\\ac_quest.tp2
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetup\\OverwriteFiles\\BWP\\BiG World Fixpack\\AC_QUEST\\ac_quest.tp2"	
@@ -214,6 +214,10 @@ LANGUAGE ~Russian~
 	 ~russian~ 
 	 ~AC_QUEST/TRA/russian/setup-AC_QUEST.tra~
 	 ~AC_QUEST/TRA/english/setup-install.tra~
+LANGUAGE ~French~ 
+	 ~french~ 
+	 ~AC_QUEST/TRA/french/setup-AC_QUEST.tra~
+	 ~AC_QUEST/TRA/french/setup-install.tra~
 
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
