--- cdtweaks\languages\french\setup.tra	
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFr\\OverwriteFiles\\BWP\\cdtweaks\\languages\\french\\setup.tra"	
@@ -1449,6 +1449,20 @@ Indiquez un pourcentage entre 20 et 500 : ~
 @330000 = ~Focus camera en cas de mort~
 @331000 = ~Demarrer les nouvelles parties avec l'IA de groupe desactivee~
 @332000 = ~Pas de depreciation dans les magasins~
+@333000 = ~Augmenter la vitesse de déplacement du groupe en dehors des combats~
+@333001 = ~de 50 pourcent~
+@333002 = ~de 100 pourcent~
+@333003 = ~de 150 pourcent~
+@333010 = ~Désactivation de hâte automatique~
+@333011 = ~Désactivation de hâte automatique
+
+Cette capacité détermine si les membres du groupe peuvent se déplacer plus rapidement en dehors des combats. Cliquez sur la capacité pour la désactiver.~
+
+@333012 = ~Activation de hâte automatique~
+@333013 = ~Activation de hâte automatique
+
+Cette capacité détermine si les membres du groupe peuvent se déplacer plus rapidement en dehors des combats. Cliquez sur la capacité pour l'activer.~
+@333020 = ~Bonus de vitesse de déplacement supprimé~
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 /////                                                  \\\\\
